akka/templates/autoscaling.yaml

```diff
@@ -25,7 +25,38 @@ spec:
     {{- toYaml .Values.autoscaling.metrics | nindent 4 }}
   minReplicas: {{ .Values.autoscaling.minReplicas }}
   scaleTargetRef:
-    kind: {{ .Values.autoscaling.scaleTargetRef.kind }}
-    apiVersion: {{ .Values.autoscaling.scaleTargetRef.apiVersion }}
+    kind: Deployment
+    apiVersion: apps/v1
     name: "{{ template "akka.fullname" . }}"
 {{- end }}
+
+{{- if .Values.autoscalingRollout.enabled -}}
```

Comment:
Do we define `autoscaling` and `autoscalingRollout` together? If they are defined together, then `---` is necessary to separate yaml, and if they are not defined together, then it would be better to add them to the condition as https://github.com/chatwork/charts/blob/master/fluentd/templates/statefulset.yaml#L2.